.ci/ci-pyp.yml

```
@@ -0,0 +1,4 @@
+pipeline_template: ci/inmanta-tools/Jenkinsfile-pyp
+repo_name: inmanta-nfv-test-api
+public: true
+build_system: pep517
```

`pep517` has been deprecated. In the pipeline it is now a synonim for "generic", which uses `python -m build`.
```suggestion
build_system: generic
```